Framework/TestHelpers/src/WorkspaceCreationHelper.cpp

```diff
@@ -112,20 +112,18 @@ Workspace2D_sptr create1DWorkspaceRand(int size, bool isHisto) {
   Counts counts(size, randFunc);
   CountStandardDeviations errorVals(size, randFunc);
 
-  auto generatedHisto = createHisto(isHisto, counts, errorVals);
   auto retVal = boost::make_shared<Workspace2D>();
-  retVal->initialize(1, std::move(generatedHisto));
+  retVal->initialize(1, createHisto(isHisto, counts, errorVals));
```

Comment:
While this eliminates the warning, it doesn't change the semantics. Is it worth adding a member function `void Mantid::API::MatrixWorkspace::initialize	(std::size_t NVectors, HistogramData::Histogram &&histogram)` so that the `Histogram` object is actually moved?